tpcx_bb/queries/load_test/tpcx_bb_load_test.py

```diff
@@ -0,0 +1,173 @@
+from xbb_tools.utils import benchmark, tpcxbb_argparser, run_dask_cudf_query
+from xbb_tools.readers import build_reader
+import os, subprocess, math, time
+
+cli_args = tpcxbb_argparser()
```

Comment:
Should we update naming conventions if these are no longer CLI args?

Reply:
Adjusted for the load_test, but will need to touch all the query files eventually as they all use `run_dask_cudf_query(cli_args=...)`.